resources/meilisearch-release.md

```diff
@@ -78,32 +78,40 @@ The steps are the same as for `How to create the first pre-release (or RC i.e. R
 
 ### How to do the official release
 
-1. Go the the GitHub interface, in the [`Release` section](https://github.com/meilisearch/meilisearch/releases).
+1. Copy/paste this TODO steps on your side
+<details>
+<summary>Release todo</summary>
+- [ ] Launch the release on the GitHub interface</br>
+- [ ] Check binaries and Docker images are ready: check the assets are attached to the release + run the Docker image on your machine to ensure it starts correctly (at least)</br>
+- [ ] Check the <a href="https://github.com/Homebrew/homebrew-core/pulls">Homebrew PR</a> has been created.</br>
+- [ ] Once everything is deployed, make a message on #ms-release to announce the new release: ping integration-team and docs-team since they should deploy their own scope now.</br>
+- [ ] Bring back commits on the `release-vX.Y.Z` into `main` (cf <a href="https://github.com/meilisearch/engine-team/blob/main/resources/meilisearch-release.md#after-the-release-bring-back-changes-to-main">this section</a>)</br>
+</details>
 
-2. Click on the `Draft a new release` button
+2. Go to the GitHub interface, in the [`Release` section](https://github.com/meilisearch/meilisearch/releases).
 
-3. Fill the form with:
+3. Click on the `Draft a new release` button
+
+4. Fill the form with:
 - The title of the release: `vX.Y.Z`
 - The tag associated to the release: should be `vX.Y.Z`
 - ⚠️ The branch on which to push the release: `release-vX.Y.Z`. Do not push the tag to `main`!
-- The description. Add as many details as possible: the PRs, the authors, new usage examples, the external contributors we want to thank...
+- The description: use the changelogs in [the related PR](https://github.com/meilisearch/engine-team/pulls). ⚠️ Some lines of the changelogs must be removed; check carefully the comments and if there is any remaining `TBD`.
 - Check the `Set as the latest release`
-- Check the `Create a discussion for these release` if necessary (not for patch releases)
 
 ![GH release form](../assets/gh-release.png)
 
 5. Finally, when you are sure, click on the `Publish release` button.
-6. If you have chosen to create a discussion, go to the newly created one and
```

Comment:
I removed this step: I checked, the created discussions have not been useful since v0.28.0 (more than a year ago) because we had a lot of breaking changes. Did not bring any interaction with users since this moment, so I would like simplify our release process when this is possible